src/test/java/org/apache/sling/models/impl/via/ChildResourceViaProviderTest.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.sling.models.impl.via;
+
+import static org.mockito.Mockito.when;
```

Comment:
Please move down to rest of import section

Reply:
done